chain/network-primitives/Cargo.toml

```
@@ -16,10 +16,11 @@ borsh = "0.9"
 serde = { version = "1", features = ["derive", "rc", "alloc"] }
 strum = { version = "0.20", features = ["derive"] }
 tracing = "0.1.13"
-
+deepsize = { version = "0.2.0", optional = true }
 near-crypto = { path = "../../core/crypto" }
-near-primitives = { path = "../../core/primitives" }
+near-primitives = { path = "../../core/primitives", features = ["deepsize_feature"] }
```

I feel we should not enable the feature unconditionally:

```suggestion
near-primitives = { path = "../../core/primitives" }
```

I see the same pattern in other Cargo.toml files, so we should address it there as well

---

Thanks. Fixed.